### PR TITLE
Docs page + cleaner start page

### PR DESCRIPTION
**Pull request by Itangalo**

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* Guidance scale: This kind of corresponds to how closely Stable Diffusion should stick to the prompt. Higher value means more strict interpretation.
+* Seed: A number between 0 and 2^32 - 1 that is used as starting point for the image generation. If the same seed is used with the same prompt, the same image will be generated. If left empty, a random seed will be used.
+
+The **History** tab show previously generated images and the prompts used. If a seed was provided, it too will be available. (If the seed setting was left empty, the random seed will _not_ be visible in the logs for version 0.3.0.)
+
+## Extra tips
+
+* For some tips for writing prompts, see for example [lexica.art](https://lexica.art/), [arthub.ai](https://arthub.ai) or ["prompt engineering" section on this page](https://www.assemblyai.com/blog/how-to-run-stable-diffusion-locally-to-generate-images/).
+* As stated in the DiffusionBee window, the application requires quite a bit of CPU power and will run faster if other applications are turned off.
+* All created images are stored at `/tmp` in your file system. The directory is wiped every time you reboot your computer.
```
Comment (dngrlux) [inline docs/readme.md:35]: * All created images are stored at `~/.diffusionbee/images/` in your file system. This hidden directory will need to be managed manually at this time.

**Comment by Itangalo** [comment]
@dngrlux: Thanks – now fixed.

File: README.md [hunk 2]
```diff
@@ -29,26 +17,22 @@ Join discord server : https://discord.gg/t6rC5RaJQn
...
-
-1) Download and start the application
-
-2) Enter the text prompt.
-<img width="1162" alt="Screen Shot 2022-09-11 at 12 33 52 PM" src="https://user-images.githubusercontent.com/1890549/189538839-45ac91b1-cd66-4543-9ece-956220d0d769.png">
-
-3) Wait a few seconds to get the generated image
-
-![image](https://user-images.githubusercontent.com/1890549/189539684-222482fb-63f7-4799-bfc1-005b84508e35.png)
+## License
```
Comment (LePips) [inline README.md:25]: Place the license as the last section.

File: README.md [hunk 3]
```diff
@@ -29,26 +17,22 @@ Join discord server : https://discord.gg/t6rC5RaJQn
...
+Stable Diffusion is released under [the CreativeML OpenRAIL M license](https://github.com/CompVis/stable-diffusion/blob/main/LICENSE).
+Diffusion Bee is just a GUI wrapper on top of Stable Diffusion, so all the term of Stable Diffusion are applied on the outputs.
 
+## How to use
```
Comment (LePips) [inline README.md:29]: Change to "Usage", this is a lot clearer and inline with a lot of other documentation I have seen and worked on.

File: docs/readme.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+## How to use DiffusionBee
+
+This documentation is written for version 0.3.0. Parts of it may be unapplicable for other versions.
```
Comment (LePips) [inline docs/readme.md:3]: Versioning is not required as it is implicit that documentation should work for the latest release version. Versioning is also kept within the git history.

File: docs/readme.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+### Installation
```
Comment (LePips) [inline docs/readme.md:5]: This section can be simplified by removing the first step, that's pretty much known by installing all applications on macOS. I would simplify to the following:

```
### Installation

On first launch, DiffusionBee will download and install additional data for generation.
```

File: docs/readme.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+* Batch size: This tells DiffusionBee to generate multiple images at a time. New batches of images will be created until "num images" have been created.
+* Guidance scale: This kind of corresponds to how closely Stable Diffusion should stick to the prompt. Higher value means more strict interpretation.
+* Seed: A number between 0 and 2^32 - 1 that is used as starting point for the image generation. If the same seed is used with the same prompt, the same image will be generated. If left empty, a random seed will be used.
+
+The **History** tab show previously generated images and the prompts used. If a seed was provided, it too will be available. (If the seed setting was left empty, the random seed will _not_ be visible in the logs for version 0.3.0.)
+
+## Extra tips
+
+* For some tips for writing prompts, see for example [lexica.art](https://lexica.art/), [arthub.ai](https://arthub.ai) or ["prompt engineering" section on this page](https://www.assemblyai.com/blog/how-to-run-stable-diffusion-locally-to-generate-images/).
+* As stated in the DiffusionBee window, the application requires quite a bit of CPU power and will run faster if other applications are turned off.
```
Comment (LePips) [inline docs/readme.md:34]:
```
* Image generation requires a substantial amount of system memory, close other applications for a faster generation.
```

File: docs/readme.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+* Batch size: This tells DiffusionBee to generate multiple images at a time. New batches of images will be created until "num images" have been created.
+* Guidance scale: This kind of corresponds to how closely Stable Diffusion should stick to the prompt. Higher value means more strict interpretation.
+* Seed: A number between 0 and 2^32 - 1 that is used as starting point for the image generation. If the same seed is used with the same prompt, the same image will be generated. If left empty, a random seed will be used.
+
+The **History** tab show previously generated images and the prompts used. If a seed was provided, it too will be available. (If the seed setting was left empty, the random seed will _not_ be visible in the logs for version 0.3.0.)
+
+## Extra tips
+
+* For some tips for writing prompts, see for example [lexica.art](https://lexica.art/), [arthub.ai](https://arthub.ai) or ["prompt engineering" section on this page](https://www.assemblyai.com/blog/how-to-run-stable-diffusion-locally-to-generate-images/).
```
Comment (LePips) [inline docs/readme.md:33]:
```
* For prompt ideas and help, check out:
  - [lexica.art](https://lexica.art/)
  - [arthub.ai](https://arthub.ai/)
  - `Prompt Engineering` of [AssemblyAI](https://www.assemblyai.com/blog/how-to-run-stable-diffusion-locally-to-generate-images/)
```

File: README.md [hunk 8]
```diff
@@ -1,24 +1,12 @@
...
-[![Download](https://user-images.githubusercontent.com/1890549/189538422-52d50488-c1fa-4924-bec6-186c9e0f307b.png)](https://diffusionbee.com/)
-
-https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/releases/download/0.3.0/DiffusionBee-0.3.0-arm64.dmg
-
-<br>
-
-For prompt ideas visit https://arthub.ai
-
-Join discord server : https://discord.gg/t6rC5RaJQn
+* Diffusion Bee is the easiest way to run Stable Diffusion locally on your M1 Mac. Comes with a one-click installer. No dependencies or technical knowledge needed.
```
Comment (LePips) [inline README.md:6]:
```
* Diffusion Bee is the easiest way to run Stable Diffusion locally on your Mac.
```

File: docs/readme.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+* As stated in the DiffusionBee window, the application requires quite a bit of CPU power and will run faster if other applications are turned off.
+* All created images are stored at `~/.diffusionbee/images/` in your file system. The directory is wiped every time you reboot your computer.
```
Comment (LePips) [inline docs/readme.md:35]:
```
* Generated images are stored at `~/.diffusionbee/images/`. The directory is wiped on system reboots.
```

File: docs/readme.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+* The model file is stored at `~/.diffusionbee/downloads`. If you already have the model available and don't want to re-download it, move, copy or symlink it to that directory.
```
Comment (LePips) [inline docs/readme.md:36]:
```
* The model is stored at `~/.diffusionbee/downloads`. If you already have the model available and don't want to re-download it, move, copy or symlink it to that directory.
```

File: docs/readme.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+* The tab _Image to Image_ is currently not used.
```
Comment (LePips) [inline docs/readme.md:37]: This is not necessary as the app itself indicates that it is not necessary.

File: docs/readme.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+* Num images: The number of images to generate.
+* Image height/width: Sets the dimensions of the image. Note that Stable Diffusion is trained on 512 x 512 (the default setting). Other dimensions may give less good result and take more time.
+* Steps: This corresponds to how many steps are used to improve the image quality. Setting to a low number gives faster image generation, and may be useful while exploring different prompts.
+* Batch size: This tells DiffusionBee to generate multiple images at a time. New batches of images will be created until "num images" have been created.
+* Guidance scale: This kind of corresponds to how closely Stable Diffusion should stick to the prompt. Higher value means more strict interpretation.
+* Seed: A number between 0 and 2^32 - 1 that is used as starting point for the image generation. If the same seed is used with the same prompt, the same image will be generated. If left empty, a random seed will be used.
+
+The **History** tab show previously generated images and the prompts used. If a seed was provided, it too will be available. (If the seed setting was left empty, the random seed will _not_ be visible in the logs for version 0.3.0.)
```
Comment (LePips) [inline docs/readme.md:29]: Remove the versioning as explained.

**Review by LePips — COMMENTED** [review]
Rename file to `DOCUMENTATION.md`.

File: docs/readme.md (hunk 9 above)
Comment (dngrlux) [reply docs/readme.md:35]: When they were stored at /tmp they were wiped at every boot, as is the nature of that system folder. Now that they are in a hidden folder within your user folder, I don't believe that applies. The new persistent history feature takes advantage of the fact that they are stored in a siloed location. This folder will accumulate images and models without any ability to manage them from the UI—the user currently has to manually clean the older models (from 0.10) and images here, as far as I know. Future enhancement needed to purge old files.

File: docs/readme.md (hunk 1 above)
Comment (Itangalo) [reply docs/readme.md]: Folder updated. I don't really understand what 'managed manually' implies, and think that it is useful to know that the directory is wiped on reboot. Leaving that part unchanged.

File: docs/readme.md (hunk 4 above)
Comment (Itangalo) [reply docs/readme.md:3]: In my experience documentation often lags behind the code, which is why I put a version reference in the docs. Leaning towards keeping it that way, but I'm happy to listen to more arguments.

**Comment by Itangalo** [comment]
Thanks @LePips and others for help on improving the documentation. I now think this is ready to be merged.

**Comment by Leland** [comment]
Normal people have no clue how to use GitHub – this should probably _eventually_ live on the website. That would actually be a great benefit for its SEO as well.

The website itself is open source in [this branch](https://github.com/divamgupta/diffusionbee-stable-diffusion-ui/tree/website) ...

But moving the documentation to it doesn't have to happen in this PR. Just getting documentation together at all is a great first step 👍

**Comment by Itangalo** [comment]
@Leland: I agree that the documentation _eventually_ should live on the website. However, the development efforts are going to the main project – not the docs – and I don't see a way to keep any documentation fairly up to date and have every update go through the project maintainer. Down the line this might be easier, when development levels out (or more maintainers are brought onboard). So if we want relevant documentation it needs to live elsewhere.

(Don't get me wrong: I'm very happy that the maintainer puts his efforts in expanding and improving the code. Putting documentation in a parallel project is a way of allowing more focus on the main project.)

**Comment by Itangalo** [comment]
And yes – most people have no clue on how to use GitHub. But I hope most people are able to start reading a README.md and following links from there. (But yeah, this is actually quite a high bar for many. If you're not used to GitHub then there are links and distracting information just about everywhere, so actually seeing the readme shouldn't be taken for granted.)

Maybe another format is better – just a Google Doc could suffice. That would allow collaboration as well, but less structured than in a GitHub project with an issue queue. On the other hand there could be an issue queue for managing larger stuff in a Google Doc.

**Comment by Itangalo** [comment]
Perhaps this conversation should move to #57 and this issue be closed…

**Comment by Itangalo** [comment]
Documentation updated for 1.1.0 and now includes img2img documentation.

**Comment by divamgupta** [comment]
Thanks for the PR, could you only add the Docs in this PR, the maybe the main README.md in a separate PR

**Comment by Itangalo** [comment]
Closing this in favour of #188 and #186.

**Pull request closed without merging**